README.md

```diff
@@ -1,22 +1,42 @@
-# dangermattic
-Shared collection of Danger plugins
+# Dangermattic
+`Dangermattic` builds on [Danger on Ruby](https://danger.systems/ruby/) and is essentially a collection of Danger plugins. Its goal is to provide customisable checks and common utilities to help perform checks on Pull Requests, from simple routine validations to more sophisticated ones.
 
 ## Installation
 
-Add to your `Gemfile`
+Add to your project's `Gemfile`
 ```
-gem 'dangermattic'
+gem 'danger-dangermattic', git: 'https://github.com/Automattic/dangermattic'
```

Comment:
Wondering if we want to keep the suggestion to point to the git repo in the README given we apparently plan to push this gem to Rubygems soon.

(We could keep this for now, but I just worry that we'll forget to remove it when comes the time that we start releasing to RubyGems?)

Reply:
Yeah, I just wanted it to reflect the current state, but indeed this will need to change eventually.
The problem is that I'm not sure when "soon" is 😅 I'd rather just keep it and have a not to change it back once we have the Rubygem published.